src/ResourceManagement/CosmosDB/Domain/IndexingPolicy/Definition/IDefinition.cs

```diff
@@ -129,6 +144,40 @@ public interface IWithCompositeIndexes<ParentT>
         /// <param name="compositePath">One of the composite path.</param>
         /// <returns>The next stage of the definition.</returns>
         IWithAttach<ParentT> WithCompositeIndex(IList<CompositePath> compositePath);
+
+        /// <summary>
+        /// Specifies a composite index empty list to set every composite path.
+        /// </summary>
+        /// <returns>The next stage of the definition.</returns>
+        IWithCompositeIndexList<ParentT> WithCompositeIndexEmptyList();
```

Comment:
Not sure, `DefineCompositeIndex` or `WithNewCompositeIndexList`?

@yaohaizh

Reply:
I'm not sure about it, too. I think using `define` for a list inside same interface is a little strange.

Reply:
I use the `WithNewCompositeIndexList`